Packs/SecurityTrails/Integrations/SecurityTrails/SecurityTrails.py

```diff
@@ -0,0 +1,411 @@
+        readable_output=tableToMarkdown(f"IPs associated with {domain}", records),
+    )
+
+    return results
+
+
+def get_domain_details(client: Client, **args) -> CommandResults:
+    domain = args.get("domain")
+    uri = f"/domain/{domain}"
+    response = client._http_request("GET", uri)
+
+    md = ""
+    current_dns = response["current_dns"]
+    del response["current_dns"]
+    md = tableToMarkdown(f"Details for {domain}", response)
+    for record_type, record_values in current_dns.items():
+        # If a record type has multiple values, this will output the last item in MD
+        temp_values = {}
+        for x in record_values["values"]:
+            temp_values.update(**x)
+        record_values.update(temp_values)
+        del record_values["values"]
+        md += tableToMarkdown(f"DNS {record_type} Records for {domain}", record_values)
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.domain_details.{domain}",
+        outputs={domain: {"domain_details": response}},
+        readable_output=md,
+    )
+
+    return results
+
+
+def get_domain_subdomains(client: Client, **args) -> CommandResults:
+    domain = args.get("domain")
+    children_only = str(args.get("children_only", "false"))
+    uri = f"/domain/{domain}/subdomains?children_only={children_only}"
+    response = client._http_request("GET", uri)
+    records = response.get("subdomains")
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.subdomains.{domain}",
+        outputs={domain: {"subdomains": records}},
+        readable_output=tableToMarkdown(
+            f"Subdomains for {domain}", records, headers="Subdomains"
+        ),
+    )
+
+    return results
+
+
+def get_domain_tags(client: Client, **args) -> CommandResults:
+    domain = args.get("domain")
+    uri = f"/domain/{domain}/tags"
+    response = client._http_request("GET", uri)
+    records = response.get("tags")
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.tags.{domain}",
+        outputs={domain: {"tags": records}},
+        readable_output=tableToMarkdown(f"Tags for {domain}", records, headers="Tags"),
+    )
+
+    return results
+
+
+def get_domain_whois(client: Client, **args) -> CommandResults:
+    domain = args.get("domain")
+    uri = f"/domain/{domain}/whois"
+    response = client._http_request("GET", uri)
+
+    results = CommandResults(
+        outputs_prefix="SecurityTrails",
+        outputs_key_field=f"SecurityTrails.whois.{domain}",
+        outputs={domain: {"whois": response}},
+        readable_output=tableToMarkdown(f"Whois for {domain}", response),
+    )
+
+    return results
+
+
+def get_domain_associated_domains(client: Client, **args) -> CommandResults:
+    domain = args.get("domain")
+
+    params = {"page": int(args.get("page"))}
```

Comment:
mypy is failing this line, Please add a default value like the rest of the commands

Reply:
added default value of 1